### PR TITLE
split agendas and minutes

### PR DESCRIPTION
**Pull request by fgregg**
In the API, eventitems combines both agenda items and items from the minutes. This makes some sense because often these items are the same. 

However, agendas and minutes are conceptually different things even if composed, in part, of the same substance and we should distinguish the two.

Since this is pretty big change, I'd like us to test this out locally.

Note that minute items are included in the eventitems even if the minutes are in draft, so right now we sometimes include items that are not intended to be published.

### CODE REVIEW

**Comment by fgregg** [comment]
Before we deploy, we should have a conversation with LA Metro about how this will change what appears on website. 

They have three options

1. Always display the agenda
2. If there are finalized minutes available, display the minutes instead of the agenda 
3. Give an option to see both agendas and minutes if both are available.

Options 1 and 2 are pretty easy,  we can just handle what we are storing as an "agenda" in the scraper.

Option 3 is a bigger lift. We would need to add a minutes property to the opencivicdata model. https://github.com/opencivicdata/docs.opencivicdata.org/issues/106 and make some medium to big sized changes to the councilmatic app.

cc @hancush @derekeder @reginafcompton

**Comment by fgregg** [comment]
Actually, looking through their data they have never finalized any minutes. It's pretty clear their preference is to to always only show the agenda.

I think we can proceed. In the future we may add minutes to other councilmatic sites.

**Comment by reginafcompton** [comment]
An addendum to the previous comment...Metro does finalize and publicize minutes for Board Meetings, although they do so in an attachment to a board report, rather than as EventItems in the Legistar API. (E.g., [this board report](http://webapi.legistar.com/v1/metro/matters/4984) points to [this attachment with meeting minutes](http://webapi.legistar.com/v1/metro/matters/4984/attachments).) 

[N.B. The Councilmatic site link to it from the event page (e.g., https://boardagendas.metro.net/event/regular-board-meeting-f9b8358d64da/).] 

In other words, Metro does make public some minutes, but not in a way that requires us to scrape them as EventItems...at least, not for the time being.

**Comment by reginafcompton** [comment]
I ran a full scrape and import of events locally. 

While this change should not affect [how Metro currently scrapes agenda items](https://github.com/datamade/scrapers-us-municipal/blob/master/lametro/events.py#L195), I saw a lot of `ERROR pupa: cannot resolve pseudo id to Bill: ~{"identifier": xxxxxxxx}`: https://gist.github.com/reginafcompton/df4371f7b02228950909c5c12161b8e5

Does this not meet your expectations @fgregg ? I expected to see far fewer of these.

**Comment by fgregg** [comment]
I'm a little confused by your comment. The number of unresolved bill with this PR should be less than or equal than the number of unresolved bills without this PR. Are you saying that the number of unresolved bills went up?

**Comment by reginafcompton** [comment]
Oh, right. I just compared: the scraper with and without the PR returns the same number of "cannot resolve" errors. (I was thinking that there should be less of such error "in general," but I do not have any reason to think that, especially with a full scrape and import on my local database.)

**Comment by fgregg** [comment]
So what is your review?
